app/src/main/java/helium314/keyboard/keyboard/Key.java

```diff
@@ -1171,13 +1171,15 @@ public KeyParams(
             if (mCode <= Constants.CODE_SPACE && mCode != KeyCode.MULTIPLE_CODE_POINTS && mIconName == null)
                 actionFlags |= ACTION_FLAGS_NO_KEY_PREVIEW;
             switch (mCode) {
-            case KeyCode.DELETE, KeyCode.ARROW_LEFT, KeyCode.ARROW_RIGHT, KeyCode.ARROW_UP, KeyCode.ARROW_DOWN:
+            case KeyCode.DELETE, KeyCode.ARROW_LEFT, KeyCode.ARROW_RIGHT, KeyCode.ARROW_UP, KeyCode.ARROW_DOWN,
+                    KeyCode.WORD_LEFT, KeyCode.WORD_RIGHT, KeyCode.PAGE_UP, KeyCode.PAGE_DOWN:
```

Comment:
This is more of a thought related to this line than an actual request for changing something:
Do you think it would be a good idea to have a reduced repeat interval for some key codes? Moving 20 words per second might be a little hard to use properly.
On the other hand, having different repeat delays can also be unexpected.

Reply:
That's a good question. I think at most 10 per second would probably be better for the larger ones.

Reply:
Thanks!
I put it on my todo list, will have a look at it soon(ish)